docs/next.config.js

```diff
@@ -78,6 +92,24 @@ module.exports = withDocsInfra({
           '.tsx',
           ...config.resolve.extensions.filter((extension) => extension !== '.tsx'),
         ],
+        alias: {
+          ...config.resolve.alias,
+          '@mui/material': path.resolve(__dirname, '../packages/mui-material/src'),
+          '@mui/docs': path.resolve(__dirname, '../packages/mui-docs/src'),
+          '@mui/icons-material': path.resolve(__dirname, '../packages/mui-icons-material/lib'),
+          '@mui/lab': path.resolve(__dirname, '../packages/mui-lab/src'),
+          '@mui/styled-engine': path.resolve(__dirname, '../packages/mui-styled-engine/src'),
+          '@mui/styles': path.resolve(__dirname, '../packages/mui-styles/src'),
+          '@mui/system': path.resolve(__dirname, '../packages/mui-system/src'),
+          '@mui/private-theming': path.resolve(__dirname, '../packages/mui-private-theming/src'),
+          '@mui/utils': path.resolve(__dirname, '../packages/mui-utils/src'),
+          '@mui/base': path.resolve(__dirname, '../packages/mui-base/src'),
+          '@mui/material-next': path.resolve(__dirname, '../packages/mui-material-next/src'),
+          '@mui/joy': path.resolve(__dirname, '../packages/mui-joy/src'),
+          docs: path.resolve(__dirname, './'),
```

Comment:
`docs` are likely not necessary to be specified here. Workspace linking should work just fine.